contactmps/static/javascript/natrepresentation.js

```diff
@@ -297,21 +290,64 @@ $(function() {
   });
 });
 
-var template = "Honourable Member {{{ recipient_name }}},\n\n{{{ content }}}\n{{{ other_issues }}}\
-\nAs a member of parliament you represent all South Africans, including me. Please vote in favour of good governance - a governance that is best suited to realising my hopes for our future.\n\nSincerely,\n{{{ sender_name }}}";
-
-function updateBody($form, recipientName) {
-  var senderName = $form.find('input[name=name]').val();
-  var letterContent = $form.find('[name=letter-content]').val();
+var template = "Hon. {{{ recipient_name }}},\n\nAs a democratically elected member of Parliament, you represent me, my concerns and my hopes for the future of South Africa.\n\nI {{{ sufficiently_represented }}}that members of Parliament represent me sufficiently as a citizen, in the National Assembly.\n\n{{{ how_voice_heard }}}.\n\nI would like to be able to raise my concerns about issues of national government with you by {{{ how_should_voice_heard }}}.\n\nMy biggest concerns about South Africa are {{{ concerns }}}.\n\nI believe it is important that I and other South Africans can have our concerns heard by national Government. I hope that we can work together to find effective ways of being heard.\n\nSincerely,\n{{{ sender_name }}}\n{{{ province }}}";
+
+function composeMessage() {
+  var sufficientlyRepresentedOption = $('#sufficiently-represented .option.selected').data('value');
+  var howVoiceHeardOptions = $('#how-voice-heard .option.selected').map(
+    function() { return $(this).data('value'); }).get();
```

Comment:
fucking jquery collections. This acts like an array but doesn't have join() and causes a DOMException because of some cross-site issue without this get()